frontend-react/src/components/Admin/AdminReceiverDashboard.tsx

```diff
@@ -547,11 +547,13 @@ function renderAllReceiverRows(props: {
                     </div>
                 </Grid>
                 <ScrollSyncPane enabled>
-                    <Grid row className={"horizontal-scroll"}>
-                        <Grid row className={"week-column"}>
-                            {perDayElements}
+                    <>
```

Comment:
Out of curiosity, why is this needed?

Reply:
A react ref error was being thrown that could only be resolved by giving the scroll panel a class-based component.

Reply:
Huh!  That's very odd...I wonder if it has something to do with [`childRef` not being used](https://github.com/okonet/react-scroll-sync/blob/master/src/ScrollSyncPane.js#L36) 🤔 

In any case, if a Fragment works for our cases (and it looks like it does), that's fine enough for me.